Python/omevv/v1/omevv_apis_client/models/job.py

```diff
@@ -190,15 +190,22 @@ def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
         next_run: Union[Unset, datetime.datetime]
         if isinstance(_next_run, Unset):
             next_run = UNSET
-        else:
-            next_run = isoparse(_next_run)
+        else:  # Added below code to handle when next_is is none
+            if _next_run is None:
+                next_run = None
+            else:
+                next_run = isoparse(_next_run)
 
         _last_run = d.pop("lastRun", UNSET)
         last_run: Union[Unset, datetime.datetime]
         if isinstance(_last_run, Unset):
             last_run = UNSET
-        else:
-            last_run = isoparse(_last_run)
+        else:  # Added below code to handle when next_is is none
```

Comment:
typo- comment to include proper variable name

Reply:
@pavithra-mahadev Corrected it.